Cargo.toml

```diff
@@ -38,6 +38,7 @@ crate-type = ["lib", "staticlib"]
 
 [dev-dependencies]
 regex = "1.9.6"
+toml = { version = "0.6.0", default-features = false, features = ["parse"] }
```

Comment:
note: the version here, and in the `Cargo.lock` was arranged based on this crate's MSRV. The `toml` stack of crates (`toml`, `toml-edit`, `serde_spanned`, etc) have aggressive MSRV reqs for newer versions. (_Another reason I didn't want them to be primary dependencies!_)